src/fides/lib/oauth/api/routes/user_endpoints.py

```diff
@@ -89,6 +89,7 @@ def create_user(
 @router.delete(
     urls.USER_DETAIL,
     status_code=HTTP_204_NO_CONTENT,
+    dependencies=[Security(verify_oauth_client, scopes=[USER_DELETE])],
```

Comment:
Just making sure the root user still passes this scope check -- the rooter user gets _every_ scope, right?